README.md

```diff
@@ -171,11 +171,40 @@ For the ECS preview variant of Bottlerocket, we recommend updating hosts using o
 
 #### Update API
 
-The [Bottlerocket API](#api) includes methods for checking and starting system updates.  You can read more about the update APIs in our [update system documentation](sources/updater/README.md#update-api).
+The [Bottlerocket API](#api) includes methods for checking and starting system updates.
+You can read more about the update APIs in our [update system documentation](sources/updater/README.md#update-api).
+
+apiclient knows how to handle those update APIs for you, and you can run it from the [control](#control-container) or [admin](#admin-container) containers.
+
+To see what updates are available:
+```
+apiclient update check
+```
+If an update is available, it will show up in the `chosen_update` field.
+The `available_updates` field will show the full list of available versions, including older versions, because Bottlerocket supports safely rolling back.
+
+To apply the latest update:
+```
+apiclient update apply
+```
+
+The next time you reboot, you'll start up in the new version, and system configuration will be automatically [migrated](sources/api/migration/).
+To reboot right away:
+```
+apiclient reboot
+```
+
+If you're confident about updating, the `apiclient update apply` command has `--check` and `--reboot` flags to combine the above actions, so you can accomplish all of the above steps like this:
+```
+apiclient update apply --check --reboot
+```
+
+See the [apiclient documentation](sources/api/apiclient/) for more details.
 
 #### Updog
 
-You can update Bottlerocket using a CLI tool, `updog`, if you [connect through the admin container](#admin-container).
+You can also update Bottlerocket using a CLI tool, `updog`, if you [connect through the admin container](#admin-container).
+This is what API-based updates use behind the scenes.
```

Comment:
Is it still worth keeping this section around? Everyone loves `updog` but it should largely be an implementation detail now.

Reply:
That's fair.  It can stick around in sources/updater/README for emergencies or special cases.  I'll update that a bit as well and post a new revision.